x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/alerts/alerts_tab_content.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import {
+  getAlertSummaryTimeRange,
+  ObservabilityAlertStatusFilter,
+  useTimeBuckets,
+} from '@kbn/observability-plugin/public';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import { InfraClientCoreStart, InfraClientStartDeps } from '../../../../../../types';
+import { useUnifiedSearchContext } from '../../../hooks/use_unified_search';
+
+import {
+  ALERTS_PER_PAGE,
+  ALERTS_TABLE_ID,
+  casesFeatures,
+  casesOwner,
+  infraAlertFeatureIds,
+} from '../config';
+import { useAlertsQuery } from '../../../hooks/use_alerts_query';
+
+export const AlertsTabContent = React.memo(() => {
+  const { services } = useKibana<InfraClientCoreStart & InfraClientStartDeps>();
+
+  const { alertStatus, setAlertStatus, alertsEsQueryByStatus } = useAlertsQuery();
+
+  const { unifiedSearchDateRange } = useUnifiedSearchContext();
+
+  const timeBuckets = useTimeBuckets();
+
+  const { application, cases, charts, triggersActionsUi } = services;
+
+  const {
+    alertsTableConfigurationRegistry,
+    getAlertsStateTable: AlertsStateTable,
+    getAlertSummaryWidget: AlertSummaryWidget,
+  } = triggersActionsUi;
+
+  const CasesContext = cases.ui.getCasesContext();
+  const uiCapabilities = application?.capabilities;
+  const casesCapabilities = cases.helpers.getUICapabilities(uiCapabilities.observabilityCases);
+
+  const summaryTimeRange = getAlertSummaryTimeRange(unifiedSearchDateRange, timeBuckets);
+
+  const chartThemes = {
+    theme: charts.theme.useChartsTheme(),
+    baseTheme: charts.theme.useChartsBaseTheme(),
+  };
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="m">
+      <EuiFlexItem>
+        <ObservabilityAlertStatusFilter onChange={setAlertStatus} status={alertStatus} />
```

Comment:
Noticed something strange with the display of the status filters in the Alerts tab vs. the Alerts view. It looks like the last item is cut off or doesn't have the appropriate style from EUI with the rounded corners.

![CleanShot 2023-02-09 at 12 32 58@2x](https://user-images.githubusercontent.com/4104278/217802019-eec74e85-9c0b-4e00-8525-273c3b6f0c21.png)

Compared to the alerts view

![CleanShot 2023-02-09 at 12 33 14@2x](https://user-images.githubusercontent.com/4104278/217801833-b9369180-4dc4-4848-8f7d-8746c7d28a9b.png)

Reply:
Good catch, just fixed it :)